db/migrations/008.do.create-participants-table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE participants (
+    id SERIAL PRIMARY KEY,
+    participant_address TEXT NOT NULL UNIQUE
+);
+
+-- Add an index for fast lookups
+CREATE INDEX idx_participant_address ON participants (participant_address);
```

Comment:
This index is redundant. Because `participant_address` is `UNIQUE`, there is already an index that can be used for fast lookups.